### ISSUE TITLE
Bad encoding leads to unaligned cells

### ISSUE THREAD

**Issue by Chrismettal** [mechatroner/vscode_rainbow_csv#103]
When the file you opened has chars that are badly encoded in it, the alignment will shift as if that char does not exist at all.

For example, the following string was saved with ANSI (Windows 1252) encoding:

```csv
This,is,a,header,with,°,bad,chars,°,sprinkled,throughout,
,,,,,,,,,,,
,,,,,,,,,,,
```

After opening it in VSCode, the default settings will try to open the file as UTF-8:

![image](https://user-images.githubusercontent.com/39636046/147951600-00701d70-8acf-4e58-992a-721aa9594f70.png)

Obviously the real problem is opening the file with the wrong encoding, but trying to align the table regardless, because sometimes you just don't care if a single symbol is decoded correctly, the alignment will be off after each bad char:

![image](https://user-images.githubusercontent.com/39636046/147951732-4f7becad-8a74-426f-8d9a-c4db0ce2e535.png)

While the problem does not start with the extension, maybe there is still a way to detect the bad char questionmark `�` during counting for alignment as well.

**Comment by mechatroner** [comment]
Thanks for reporting. I agree that this is an issue, and this might actually be a special case of a more general problem, see #69

**Comment by Chrismettal** [comment]
You are right, I thought it was an issue of the <?> not being counted when padding other cells, but showing whitespace reveals that <?> just does not adhere to the fixed with font width like your referenced issue.

![image](https://user-images.githubusercontent.com/39636046/150816339-9b95b221-9643-47d8-8da4-43610a506dbd.png)

**Comment by mechatroner** [comment]
I fixed #69 and it also resolves this issue, although alignment might still be a little bit "wobbly" which can be explained by font rendering issues: https://stackoverflow.com/a/7221188/2898283
